portal/views/partials/header/header.njk

```diff
@@ -1,6 +1,6 @@
 <nav class="navbar navbar-expand-lg navbar-static-top">
   <div class="container">
-    <a class="navbar-brand mobile-hide-large header__icon" href="{{ baseUrl }}/"></a>
```

Comment:
May I ask why do we need to remove `mobile-hide-large`?

Reply:
![rsz_localhost_3000_iphone_x](https://user-images.githubusercontent.com/19148531/70350858-49b19500-1870-11ea-80b2-c65e346eeecb.png)

I was following the design, making the logo visible on mobile, i can add back the class if that's what you would like

Reply:
Ah, right. Thanks @mul53

Reply:
So i shouldn't add back the class

Reply:
yeah. let' remove that class.

Reply:
okay then